vnext/Microsoft.ReactNative.Cxx/NativeModules.h

```diff
@@ -4,33 +4,40 @@
 #pragma once
 #include "winrt/Microsoft.ReactNative.h"
 
-#include <type_traits>
 #include "JSValueReader.h"
 #include "JSValueWriter.h"
-#include "ModuleMemberRegistration.h"
 #include "ModuleRegistration.h"
 #include "ReactPromise.h"
 
-// REACT_MODULE annotates a C++ struct as a ReactNative module.
-// It can be any struct which can be instantiated using a default constructor.
-// Note that it must be a 'struct', not 'class' because macro does a forward declaration using the 'struct' keyword.
-//
+#include <type_traits>
+
+// REACT_MODULE(moduleStruct, [opt] moduleName, [opt] eventEmitterName)
 // Arguments:
 // - moduleStruct (required) - the struct name the macro is attached to.
 // - moduleName (optional) - the module name visible to JavaScript. Default is the moduleStruct name.
 // - eventEmitterName (optional) - the default event emitter name used by REACT_EVENT.
 //   Default is the RCTDeviceEventEmitter.
+//
+// REACT_MODULE annotates a C++ struct as a ReactNative module.
```

Comment:
>REACT_MODULE [](start = 3, length = 12)

Nit: Given how overloaded terms like MODULE and INIT are, should we have more specific macro names? Perhaps encode the relation to native modules in the macro prefix somehow (e.g. "REACTNM_" instead of "REACT_")?